examples/react-native/package.json

```diff
@@ -34,7 +33,6 @@
     "@storybook/addon-ondevice-actions": "^5.3.23",
     "@storybook/addon-ondevice-knobs": "^5.3.26",
     "@storybook/react-native": "^5.3.25",
-    "@storybook/react-native-server": "^5.3.23",
```

Comment:
We would want to remove the `react-native-storybook-loader` dependency at line 47 below, not the rn-server.